rally_ovs/plugins/ovs/ovsclients.py

```diff
@@ -101,6 +101,11 @@ def get_lswitch_info(info):
 
     return lswitches
 
+def get_lswitch_by_name(lswitch_data):
```

Comment:
function named as ..._by_name usually means the input is a name, and it returns the results for the given name. This function is for parsing ls-list output, so please rename it to parse_lswitch_list.

Reply:
ok will do

Reply:
done